update to latest mdht version

LakshmiDintakurty opened the pull request:
Updated to latest MDHT version

LakshmiDintakurty committed:
update to latest mdht version

HaiwenZhuCerner commented on pom.xml:
```diff
@@ -45,62 +42,51 @@
+		   <!-- <version>2.6.3.20170323</version> -->
+		   <version>3.0.8.20180531</version>	
 		</dependency>
 		
```
I still think we should add below 2 jars that's from recent MDHT releases and ETT Validator releases:

        <dependency>
            <groupId>mdht.hl7.ds4p</groupId>
            <artifactId>org.hl7.cbcc.privacy.consentdirective</artifactId>
            <version>${consentdirective.version}</version>
        </dependency>
        <dependency>
            <groupId>mdht.hl7.ds4p</groupId>
            <artifactId>org.hl7.security.ds4p.contentprofile</artifactId>
            <version>${ds4pcontent.version}</version>
        </dependency>

ETT Validator's referenceccdavalidator/pom.xml has them: 
https://github.com/siteadmin/reference-ccda-validator/blob/master/pom.xml#L64

MDHT official releases have these 2 relatively new jars in the mdht folder, e.g.: MDHT_CCDA_R2.1_R1.1_MU2_DS4P_20180531_build\mdhtruntime\mdht

Your updated ais-validator also has these 2 jars, see 
https://github.cerner.com/ais/ais-validator/blob/master/dependencies.gradle#L80 

I understand our referenceccdavalidator is not using the features in these 2 jars yet.  Still, since we are updating the MDHT jars set, it makes sense to include all jars from MDHT that our reference ETT Validator (11/27/2018 ETT Validator release v1.0.37) has.  Thank you.

LakshmiDintakurty replied:
added the 2 jars

HaiwenZhuCerner reviewed (commented):
Looks good, thank you Lakshmi.  I highly recommend you also include in this referenceccdavalidator/pom.xml the 2 new mdhtruntime jars that ETT reference-ccda-validator/pom.xml has and our ais-validator's dependencies.gradle also has (refer below).  I have a more detailed comment in the code below.

 'org.hl7.security.ds4p.contentprofile:org.hl7.security.ds4p.contentprofile:3.0.0.20170920',			   'org.hl7.cbcc.privacy.consentdirective:org.hl7.cbcc.privacy.consentdirective:1.0.0.20170920'

LakshmiDintakurty committed:
add the two jars
org.hl7.cbcc.privacy.consentdirective-1.0.0.20170920.jar
org.hl7.security.ds4p.contentprofile-3.0.0.20170920.jar

HaiwenZhuCerner reviewed (approved):
+1 cool

LakshmiDintakurty committed:
change to secure

LakshmiDintakurty commented:
Fixed secure repo url

HaiwenZhuCerner reviewed (approved):
+1 Cool Lakshmi, what else did you change to make the https Cerner Artifactory URL work?
